src/util/types.ts

```diff
@@ -584,7 +584,6 @@ export const SVGElementPropKeys = [
   'name',
   'style',
   'target',
-  'type',
```

Comment:
Please do add a comment as to why type is not used here. 
Otherwise another engineer might one day add this back, arguing `Well type exists as a prop, see https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/type`.

And yes, it does, but it conflicts with an existing recharts prop, thus we do not allow it.

Reply:
> conflicts with an existing recharts prop, thus we do not allow it.

I don't think this is quite the right way to frame it as we do have some conflicting props that we have to allow (can't think off the top of my head but I know they're there). More like we can remove it until there is a use-case for an SVG element that utilizes the attribute (if ever).

Not sure that adding comments inline for every removed attribute is sustainable... I would love to utilize that SVG attribute type library I have mentioned in the past. But will add something here for now